colcon_bundle/verb/bundlefile.py

```diff
@@ -28,7 +28,11 @@ def __init__(self, *, name=None, mode='w'):
         :param name: path to the bundle archive
         :param mode: mode to open file as
         """
-        self.tarfile = tarfile.open(name, mode)
+        with _set_temporary_tarfile_default_format(tarfile.GNU_FORMAT):
+            self.tarfile = tarfile.TarFile(
+                name=name, mode=mode, format=tarfile.DEFAULT_FORMAT)
```

Comment:
Why are we switching formats here (GNU to default?)

Reply:
We're setting the default to GNU and then using it to initialize.

Reply:
Got it, I should've realized the default_format was changed using the context manager.